bombast/__init__.py

```diff
@@ -135,6 +135,27 @@ def visit_ClassDef(self, node):
             return ClassDef(name, bases,
                             node.keywords, body, decorator_list)
 
+    def visit_JoinedStr(self, node):
+        new_node = None
+        for value in node.values:
+            if type(value) == Constant:
+                value = self.visit(value)
+            elif type(value) == FormattedValue:
+                value.value = self.visit(value.value)
+                value = JoinedStr(values=[value])
+            else:
+                raise RuntimeError('Invalid node in JoinedStr')
```

Comment:
This error is raised on 3.6 and 3.7, since we're explicitly checking whether formatted values are `Constant`s. It's fine to fail on those versions, but I think it's avoidable if we make this slight refactor to avoid explicit type-checking.